lib/ts/recipe/session/index.ts

```diff
@@ -346,50 +343,41 @@ export default class SessionWrapper {
         return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.getAllSessionHandlesForUser({
             userId,
             tenantId: tenantId === undefined ? DEFAULT_TENANT_ID : tenantId,
+            fetchAcrossAllTenants: tenantId === undefined,
             userContext,
         });
     }
 
-    static revokeSession(sessionHandle: string, tenantId?: string, userContext: any = {}) {
+    static revokeSession(sessionHandle: string, userContext: any = {}) {
         return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.revokeSession({
             sessionHandle,
-            tenantId: tenantId === undefined ? DEFAULT_TENANT_ID : tenantId,
             userContext,
         });
     }
 
-    static revokeMultipleSessions(sessionHandles: string[], tenantId?: string, userContext: any = {}) {
+    static revokeMultipleSessions(sessionHandles: string[], userContext: any = {}) {
         return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.revokeMultipleSessions({
             sessionHandles,
-            tenantId: tenantId === undefined ? DEFAULT_TENANT_ID : tenantId,
             userContext,
         });
     }
 
-    static updateSessionDataInDatabase(
-        sessionHandle: string,
-        newSessionData: any,
-        tenantId?: string,
-        userContext: any = {}
-    ) {
+    static updateSessionDataInDatabase(sessionHandle: string, newSessionData: any, userContext: any = {}) {
         return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.updateSessionDataInDatabase({
             sessionHandle,
             newSessionData,
-            tenantId: tenantId === undefined ? DEFAULT_TENANT_ID : tenantId,
             userContext,
         });
     }
 
     static mergeIntoAccessTokenPayload(
         sessionHandle: string,
         accessTokenPayloadUpdate: JSONObject,
-        tenantId?: string,
         userContext: any = {}
     ) {
         return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.mergeIntoAccessTokenPayload({
             sessionHandle,
             accessTokenPayloadUpdate,
-            tenantId: tenantId === undefined ? DEFAULT_TENANT_ID : tenantId,
             userContext,
         });
     }
```

Comment:
fetchAndSetClaim, setClaimValue, getClaimValue, removeClaim below should not take a tenantId either.

Reply:
Done